### PR TITLE
fix candra

### PR DESCRIPTION
**Pull request by Shreeshrii**

### CODE REVIEW

**Comment by akprasad** [comment]
Looking at this now. Thanks for your patience.

File: README.md [hunk 1]
```diff
@@ -71,3 +71,19 @@ Adding a new scheme is simple:
...
 For help in creating `schemeData`, see the comments on the `addBrahmicScheme` and `addRomanScheme` functions.
+
+Testing
+-------
+
+Top level `Makefile` Starts a Webserver using `PHP` on Port `9732`.
+
+`make` - will start the webserver on port `9732`
+
+Open your browser and launch this url `http://127.0.0.1:9732/test/index.html` and see that all tests are ok.
```
Comment (akprasad) [inline README.md]: I don't see why this is necessary. Can you explain? The `localhost`/qunit approach that exists already seems fine.

File: README.md [hunk 2]
```diff
@@ -71,3 +71,19 @@ Adding a new scheme is simple:
...
+
+Changelog
```
Comment (akprasad) [inline README.md]: No need for changelog, Git tracks that all already.

File: sanscript/sanscript.js [hunk 3]
```diff
@@ -5,14 +5,29 @@
  * other Indian languages only incidentally.
  *
  * Released under the MIT and GPL Licenses.
+ *
```
Comment (akprasad) [inline sanscript/sanscript.js]: See comment above about changelog.

File: sanscript/sanscript.js [hunk 4]
```diff
@@ -33,16 +48,56 @@
...
-            symbols: '০ ১ ২ ৩ ৪ ৫ ৬ ৭ ৮ ৯ ॐ ঽ । ॥'.split(' '),
-            other: '    ড ঢ  য '.split(' ')
+            consonants: 'ক খ গ ঘ ঙ চ ছ জ ঝ ঞ ট ঠ ড ঢ ণ ত থ দ ধ ন প ফ ব ভ ম য় র ল ব শ ষ স হ ল় ক্ষ জ্ঞ'.split(' '),
+            symbols: '০ ১ ২ ৩ ৪ ৫ ৬ ৭ ৮ ৯ ওঁ ঽ । ॥'.split(' '),
+            other: '    ড ঢ  য ল'.split(' '),
+            candra: [''],
+            accent: ['\u2060', '\u2060', "\u2060", "\u2060"],
+            combo_accent: ["", "", "", ""]
+        },
+		/* Grantha
```
Comment (akprasad) [inline sanscript/sanscript.js]: nit: line up comment with ones below.

**Comment by Shreeshrii** [comment]
Arun,

The Readme changes are by Naresh, I had forked his repo for use with
sanskrit documents.

Not being familiar with git, GitHub I was keeping a changelog within the js
script for my reference.

I think it would be easiest if you can include the relevant changes in your
repo and ignore what is unnecessary.

Thanks!

**Comment by Shreeshrii** [comment]
If you think it will help, I can close this PR.

You can suggest how to proceed after that.

**Comment by akprasad** [comment]
Thanks! I'll close this pull request and merge everything in tomorrow.

**Comment by Shreeshrii** [comment]
Thanks. Please also make the changes to sanscript.php.

**Comment by akprasad** [comment]
Taking a little longer than I expected after merging in another PR. But I'll do this soon (within a week or so).

**Pull request closed without merging**